### DISCUSSION TITLE
После включения пропадает интернет

### DISCUSSION THREAD

**Discussion by dovecoteescapee** [dovecoteescapee/ByeDPIAndroid#77] [Q&A]
В #53 и не только сообщали, что после включения пропадает интернет. Кто столкнулся с этой проблемой:
1. Установите [версию 1.0.2](https://github.com/dovecoteescapee/ByeDPIAndroid/releases/tag/1.0.2)
2. Включите в настройках `Use command line settings`. Менять в `Command line editor` ничего не надо (по умолчанию, пустая строка, при таких настройках не применяются методы обхода блокировок)
3. Запустите ByeDPI в режиме VPN
4. Проверьте работу сети и сообщите, сохраняется ли проблема

**Comment by mortonlame** [comment]
Not working anyway

**Reply by Denchik2024** [reply]
К сожалению, не помогло.

**Reply by dovecoteescapee** [reply]
Работает ли на других устройствах этой сети? Работает ли в другой сети (например, если переключить с мобильного интернета на Wi-Fi)?

**Reply by Viktor45** [reply]
> Not working anyway

After starting app in VPN mode on AndroidTV, if you got "network disconnected" message - try to go to the "network settings" of AndroidTV and re-request DHCP-address to bring back network connectivity, is it helps?

**Comment by dovecoteescapee** [comment]
Судя по [этому сообщению](https://github.com/dovecoteescapee/ByeDPIAndroid/pull/86#issuecomment-2289228331) проблема с UDP. Сообщите, пожалуйста:
1. Наблюдается ли проблема в версии 1.0.2 при следующих настройках? (UDP идёт через прокси, но к нему не применяются методы запутывания)
   ```plaintext
   Desync HTTP: true
   Desync HTTPS: true
   Desync UDP: false
   ``` 
2. Наблюдается ли проблема в версии 1.0.1? (UDP идёт напрямую)

**Reply by dovecoteescapee** [reply]
@Viktor45 можете, пожалуйста, провести тест на своём телевизоре? Я пытаюсь отладить эту проблема, но пока не уверен, где она возникает

**Reply by Viktor45** [reply]
Что-то похожее я вижу на AndroidTV версии 9 (API 28) и ниже

**Reply by qq-spirit** [reply]
> @Viktor45 можете, пожалуйста, провести тест на своём телевизоре? Я пытаюсь отладить эту проблема, но пока не уверен, где она возникает

Для особо одаренных можно переформулировать? Нужно попробовать обе версии программы и сравнить результат?

**Reply by dovecoteescapee** [reply]
> Для особо одаренных можно переформулировать? Нужно попробовать обе версии программы и сравнить результат?

Да, мне важно узнать, в обоих ли версиях есть проблема (при указанных настройках)

**Reply by qq-spirit** [reply]
> > Для особо одаренных можно переформулировать? Нужно попробовать обе версии программы и сравнить результат?
> 
> Да, мне важно узнать, в обоих ли версиях есть проблема (при указанных настройках)

У меня и на 01 и 02 версиях одинаковая ошибка. Можешь в ТГ написать @jewelhux, оперативно потестим, посмотрим если что нужно будет

**Comment by OlgrieD** [comment]
Такая же проблема (AndroidTV)  , аналогично и с protonvpn и openvpn. Octohide VPN кстати работает

**Comment by GiBroniPro** [comment]
Устройство: Samsung A23.
Столкнулся с похожей проблемой. После включения, интернет не пропадает, во всех других приложениях работает нормально, но заходя в ютуб, видео не грузит. На мобильном интернете работает плохо, на wi-fi воьще не грузит видео

**Reply by dovecoteescapee** [reply]
Как я понимаю, у вас просто не работает обход блокировки ютуба, это не та же проблема

**Comment by dmyshab** [comment]
Приветствую. Схожая проблема, сейчас опишу.
На телефоне Samsung S21 с моим домашним интернет-провайдером, работает идеально, открываются все заблоченный сайты и приложения, включая инсту. Версия андроида - 14.
А вот на АндроидТВ 9-ой версии не работает совсем. Если использовать ethernet, то тупо отрубает сеть, через вайфай, интернет-подключение также перестаёт подавать признаки жизни, но никак об этом не сигнализирует. Может быть, есть какое-то решение для 9-го андроида?

**Comment by emestee** [comment]
На  Yandex TV (YNDX-0072) наблюдается описываемое явление. 

С ByeDpi 1.0.2 с установками по умолчанию при включении byedpi в режиме VPN немедленно отваливается сеть. Лечится перезапросом DHCP или вытаскиванием кабеля. При переключении в режим прокси проблема исчезает.

При запуске Youtube c desync http + desync https приложение зависает с чёрным экраном. При включении опции desync udp приложение загружается, но, судя по всему, замедление ютуба в действии - заглавные картинки загружаются очень медленно, проигрывание видео зависает. По этому поводу в GoodbyeDPI написан комментарий: https://github.com/ValdikSS/GoodbyeDPI/issues/378, необходимо включить специфичный для России блэклист, который присутсвует в 0.2.3rc3 и в вашей сборке его, надо полагать, нет.

Поэтому здесь ДВЕ раздельные проблемы: 1) на некоторых андроидах при включении в режиме VPN отваливается DHCP 2) проблема с новым замедлением ютуба
